ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -983,10 +1031,14 @@ def runTest(self):
             thr.start()
 
             self.wait_until_reboot()
-            if self.reboot_type == 'fast-reboot':
-                self.handle_fast_reboot_health_check()
-            if self.reboot_type == 'warm-reboot':
-                self.handle_warm_reboot_health_check()
+            if self.kvm_test:
+                self.handle_advanced_reboot_health_check_kvm()
```

Comment:
I see you have separate health check for kvm and so suggest to clean up the code a bit as there is not need to check for kvm_test inside `handle_fast_reboot_health_check` and `handle_warm_reboot_health_check`.

Reply:
Cleaned up as suggested.